src/index.html

```diff
@@ -1,14 +1,35 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Snake</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Snake</h1>
-</body>
+  <head>
+    <meta
+      charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width,
+               user-scalable=no,
+               initial-scale=1.0,
+               maximum-scale=1.0,
+               minimum-scale=1.0"
+    >
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
+    <title>Snake</title>
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    >
+  </head>
+  <body>
+    <!-- <h1>Snake</h1> -->
+    <div class="snake_case">
+      <nav class="column column--1">1</nav>
```

Comment:
```suggestion
      <nav class="column">1</nav>
```
Тебе хватит одного класса для стилизации, цвет можешь задавать через nth-child